flow/SystemMonitor.cpp

```diff
@@ -110,7 +110,8 @@ SystemStatistics customSystemMonitor(std::string eventName, StatisticsState *sta
 				.DETAILALLOCATORMEMUSAGE(512)
 				.DETAILALLOCATORMEMUSAGE(1024)
 				.DETAILALLOCATORMEMUSAGE(2048)
-				.DETAILALLOCATORMEMUSAGE(4096);
+				.DETAILALLOCATORMEMUSAGE(4096)
```

Comment:
Is there an 8192 that we should be logging here?

Reply:
yup